data-connector-example/dataconnectors/data-connector-1/actions/import.ts

```diff
@@ -0,0 +1,30 @@
+import {DataConnectorAsynchronousAction, SerializedFields} from 'lucid-extension-sdk';
+import { ApiFolder, convertToFolder, folderSchema } from '../collections/folders';
```

Comment:
Nit: different formatting styles for your imports

Reply:
Added prettier so these are consistent